kouncil-frontend/package.json

```diff
@@ -4,6 +4,7 @@
   "license": "MIT",
   "scripts": {
     "postinstall": "ngcc",
+    "nx": "nx",
```

Comment:
cos to zmienia?

Reply:
Hmm, bez tego mi nie działało, ale widzę że już działa